.github/workflows/changelog-and-release.yml

```diff
@@ -16,19 +16,19 @@ on:
       version_type:
         description: 'Create a ''minor'' or ''micro'' release?'
         required: true
-        default: 'micro'
+        default: 'minor'
       changelog_text:
         description: 'Input the changes you''d like to add to the changelogs. Your text should be encapsulated in "''s with line feeds represented by literal \n''s. ie. "This is the first change\nThis is the second change"'
         required: true
         default: ''
       update_news:
         description: 'Update news in addon.xml.in? [true|false]'
         required: true
-        default: 'false'
+        default: 'true'
```

Comment:
this should be always false news are not in the .txt file

Reply:
Changed as requested.